### DISCUSSION TITLE
Change wlogout menu template

### DISCUSSION THREAD

**Discussion by kona6zug** [prasanthrangan/hyprdots#1161] [General]
Hello guys, 
i wanted to ask you how i can change the wlogout menu to the style 2 so i can see the options hiberation suspend stc.
Thank you for all your help

**Comment by mahaveergurjar** [comment]
Already this option is added you can see this in key hint or press super + backspace to see the logout menu with suspend and hibernation

**Comment by kRHYME7** [comment]
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/48f5a36d-34ed-4a0c-ad52-7dbacd13f870)

Are you pertaining to the power icon? 

right click on it to have layout 1

```
❯ ~/.config/hyprdots/scripts/logoutlaunch.sh 1
❯ ~/.config/hyprdots/scripts/logoutlaunch.sh 2
```
